examples/https/package.json

```
@@ -30,14 +33,14 @@
   },
   "dependencies": {
     "@opentelemetry/api": "^1.0.2",
```

```suggestion
    "@opentelemetry/api": "^1.0.0",
```

feel free to ignore as it is not directly related to this change. But usually we refer the lowest possible version.

---

done in [9669686](https://github.com/open-telemetry/opentelemetry-js/pull/3152/commits/966968611990160560ec009b15fc10d24e60cce8). Thanks! :slightly_smiling_face: